app/controllers/dashboard_controller.rb

```diff
@@ -9,6 +9,8 @@ def my_tracks
   end
 
   def track_library
+    # TODO: make users only the team members of the manager
+    @users = User.all
```

Comment:
Maybe better filter the users here rather than in the view.

Reply:
done